src/workloads/execution/ColumnstoreIndexes.yml

```diff
@@ -0,0 +1,463 @@
+SchemaVersion: 2018-07-01
+Owner: "@mongodb/query-execution"
+Description: |
+  This workload compares the query performance with columnar indexes to performances without them.
+
+  The workload consists of the following phases and actors:
+    0. Warm up cache.
+    1. Run queries with columnstore indexes.
+    2. Drop columnstore indexes.
+    3. Warm up cache again for regular scans.
+    4. Run queries without columnstore indexes.
+
+Keywords:
+- columnstore
+- analytics
+
+# Defines the database name.
+Database: &db charts-metrics
+
+# Defines the collection name.
+Collection: &Collection events
+
+CountPerActionQuery: &CountPerActionQuery
+  aggregate: *Collection
+  pipeline: [{"$group": {"_id": "$action", o: {$sum: 1}}}, {"$limit": 5000}]
+  allowDiskUse: true
+  cursor: {batchSize: 5000}
+
+TotalTenantCreatedQuery: &TotalTenantCreatedQuery
+  aggregate: *Collection
+  pipeline:
+    [
+      {"$match": {"metadata.target": "cloud"}},
+      {"$group": {"_id": {}, "__alias_0": {"$addToSet": "$metadata.tenant_id"}}},
+      {"$project": {"_id": 0, "__alias_0": {"$size": "$__alias_0"}}},
+      {"$project": {"value": "$__alias_0", "_id": 0}},
+      {"$limit": 5000}
+    ]
+  allowDiskUse: true
+  cursor: {batchSize: 5000}
```

Comment:
Regarding the batch size, I do think this is not a good pattern to use for future testing or for larger data sets. As we discussed on slack:
> What I am worried about is that we are not exhausting the cursor. It is related to batch size but I don't think can be fixed by only specifying or not specifying that parameter. 
We are only generating aggregate commands in this workload and never getMore commands, so there is no guarantee that we see all results even if we match the batch size to the limit, because
> To clarify why it can't be fixed with batch size: the batch size is an upper bound on each batch. Fundamentally the wire protocol uses BSON and BSON is capped at 16MB. If you request 50,000 things but we can only fit 10,000 then we will send 10,000 and a cursorId

Could you please file a follow-up ticket to either introduce some actors that can exhaust cursors or do what we do in some other workloads like [add a trailing $skip](https://github.com/mongodb/mongo-perf/blob/905cd5185fc520d6952a292b04ecb3bddee2c2bb/testcases/pipelines.js#L94-L128) or [use explain to execute the query](https://github.com/mongodb/genny/commit/6ccaf2baa6b030d731bec31a36310d1964fd64c9) - looks like that was not favored and phased out actually. 

This does seem to be a more pervasive problem, as I notice a bunch of tests using the batchSize in this way.

Reply:
Thanks for suggestion.
Filed https://jira.mongodb.org/browse/PERF-3066